BUILD

```diff
@@ -17,36 +17,13 @@
 # under the License.
 #
 
-load("@io_bazel_stardoc//stardoc:stardoc.bzl", "stardoc")
+load("@bazel_stardoc//stardoc:stardoc.bzl", "stardoc")
```

Comment:
To match one of our dependencies' use of `bazel_stardoc`, so we don't end up with `io_bazel_stardoc` and `bazel_stardoc` with different names in the same workspace, we change the name in our workspace.